config/deploy.example.rb

```diff
@@ -6,98 +6,70 @@
 # `cap deploy` whenever you would like to deploy Errbit. Refer
 # to the Readme for more information.
 
-config = YAML.load_file('config/config.yml')['deployment'] || {}
+# config valid only for current version of Capistrano
+lock '3.3.5'
 
-require 'bundler/capistrano'
-load 'deploy/assets'
+config = YAML.load_file(File.expand_path('../config.yml', __FILE__))['deployment'] || {}
 
-set :application, "errbit"
-set :repository,  config['repository']
-
-role :web, config['hosts']['web']
-role :app, config['hosts']['app']
-role :db,  config['hosts']['db'], :primary => true
-
-set :user, config['user']
-set :use_sudo, false
-if config.has_key?('ssh_key')
-  set :ssh_options,      { :forward_agent => true, :keys => [ config['ssh_key'] ] }
-else
-  set :ssh_options,      { :forward_agent => true }
-end
-default_run_options[:pty] = true
-
-set :deploy_to, config['deploy_to']
-set :deploy_via, :remote_cache
-set :copy_cache, true
-set :copy_exclude, [".git"]
-set :copy_compression, :bz2
-
-set :scm, :git
-set :scm_verbose, true
+set :application, 'errbit'
+set :repo_url, config['repo_url']
 set :branch, config['branch'] || 'master'
+set :deploy_to, config['deploy_to']
+set :keep_releases, 5
 
-before 'deploy:assets:symlink', 'errbit:symlink_configs'
-# if unicorn is started through something like runit (the tool which restarts the process when it's stopped)
-# after 'deploy:restart', 'unicorn:stop'
-
-namespace :deploy do
-  task :start do ; end
-  task :stop do ; end
-  task :restart, :roles => :app, :except => { :no_release => true } do
-    run "#{try_sudo} touch #{File.join(current_path,'tmp','restart.txt')}"
-  end
+set :pty, true
+if config.key?('ssh_key')
+  set :ssh_options, forward_agent: true, keys: [ config['ssh_key'] ]
+else
+  set :ssh_options, forward_agent: true
 end
 
-namespace :errbit do
-  task :setup_configs do
```

Comment:
Added back except secret_token.rb creation.

I'm using rbenv and calling bundle exec inside script doesn't work very well. Once #778 is merged, we will not need to handle `secret_token.rb`, so I remove it.